collectors/cronjob_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"testing"
 	"time"
 
-	v2batch "k8s.io/api/batch/v2alpha1"
+	v2batch "k8s.io/api/batch/v1beta1"
```

Comment:
Can you fix these import renames? v2 is now off.

Reply:
@brancz  hm do you mean v2beta1? Can not find this version in the kubernetes docs

Reply:
I mean `v2batch` should be `batchv1beta1`.